File: easy-solution/galaxy4276/35.rs
impl Solution {
  pub fn search_insert(nums: Vec<i32>, target: i32) -> i32 {
        for i in 0..nums.len() {
          if nums[i] == target {
            return i as i32;
          }
        }

        for i in 0..nums.len() {
          if nums[i] > target {
            return i as i32;
          }
        }

        return nums.len() as i32; 
  }   
}

File: easy-solution/minesp3164/python/35.py
class Solution(object):
    def searchInsert(self, nums, target):
        count = 0
        for i in range(len(nums)):
            if(nums[i] < target):
                count = count + 1
            else:
                break
        return count
        """
        :type nums: List[int]
        :type target: int
        :rtype: int
        """
